client/signup/jetpack-connect/sso.jsx

```diff
@@ -44,18 +46,15 @@ import EmailVerificationGate from 'components/email-verification/email-verificat
  */
 const debug = debugModule( 'calypso:jetpack-connect:sso' );
 
-const JetpackSSOForm = React.createClass( {
-	displayName: 'JetpackSSOForm',
+class JetpackSSOForm extends Component {
```

Comment:
One thing about naming here: according to our [JavaScript Coding Guidelines: Naming Conventions](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#naming-conventions) this one should be `JetpackSsoForm` instead of `JetpackSSOForm`.